crates/metaslang/cst/src/query/parser.rs

```diff
@@ -289,10 +325,13 @@ fn text_token(i: &str) -> IResult<&str, String, VerboseError<&str>> {
     .parse(i)
 }
 
-fn ellipsis_token(i: &str) -> IResult<&str, &str, VerboseError<&str>> {
-    terminated(tag("..."), multispace0).parse(i)
-}
-
 fn token<'input>(c: char) -> impl Parser<&'input str, char, VerboseError<&'input str>> {
     terminated(char(c), multispace0)
 }
+
+fn anchor<T: KindTypes>(i: &str) -> IResult<&str, ASTNode<T>, VerboseError<&str>> {
+    // An anchor is a single '.' character, and cannot be followed by another anchor
+    pair(token('.'), cut(peek(none_of(". \t\r\n"))))
```

Comment:
IIRC, we discussed still parsing the ellipsis operator, but producing a specific error explaining that it has been deprecated, and how to use anchors instead. How is that handled in the new code?